ctc_lib/container_tool_command.go

```diff
@@ -41,9 +41,9 @@ Either implement Command.Run implementation or RunO implemetation`)
 	return nil
 }
 
-func (ctc *ContainerToolCommand) PrintO(c *cobra.Command, args []string) {
+func (ctc *ContainerToolCommand) PrintO(c *cobra.Command, args []string) error {
```

Comment:
instead of `PrintO` can this be `PrintOutput` or something similar? I'm actually not totally sure what the O stands for here. a one line description would be helpful as well

Reply:
We wanted this name to allign with other Run functions of Cobra like RunE (Runs and returns Error)
RunO and PrintO are returns Object. Will add Documentation,